x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/schema_table.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiTable,
+  EuiTableHeader,
+  EuiTableHeaderCell,
+  EuiTableBody,
+  EuiTableRow,
+  EuiTableRowCell,
+  EuiHealth,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { SchemaFieldTypeSelect } from '../../../../shared/schema';
+import { FIELD_NAME, FIELD_TYPE } from '../../../../shared/schema/constants';
+
+import { SchemaLogic } from '../schema_logic';
+
+export const SchemaTable: React.FC = () => {
+  const { schema, unconfirmedFields } = useValues(SchemaLogic);
+  const { updateSchemaFieldType } = useActions(SchemaLogic);
+
+  return (
+    <EuiTable tableLayout="auto">
+      <EuiTableHeader>
+        <EuiTableHeaderCell>{FIELD_NAME}</EuiTableHeaderCell>
+        <EuiTableHeaderCell aria-hidden />
+        <EuiTableHeaderCell align="right">{FIELD_TYPE}</EuiTableHeaderCell>
+      </EuiTableHeader>
+      <EuiTableBody>
+        <EuiTableRow style={{ height: 56 }}>
+          <EuiTableRowCell>
+            <EuiText color="subdued">
+              <code>id</code>
```

Comment:
Note: This is deliberately **not** i18n'd. id is a static lowercase string set & sent back from the server-side and should not be changed or translated.